client/src/elm/Pages/WellChildProgressReport/View.elm

```diff
@@ -34,7 +34,7 @@ import Backend.WellChildEncounter.Model
 import Date
 import EverySet exposing (EverySet)
 import Gizra.Html exposing (emptyNode)
-import Gizra.NominalDate exposing (NominalDate, diffMonths, diffWeeks, formatDDMMYY, formatDDMMyyyy)
+import Gizra.NominalDate exposing (NominalDate, diffMonths, diffWeeks, formatDDMMyyyy)
```

Comment:
Please add screenshots for changes at this file.
Since date becomes longer, we may need to adjust the styling.